include/pcmki/pcmki_transition.h

```diff
@@ -130,7 +130,7 @@ typedef struct {
 
 
 typedef struct {
-    gboolean (*pseudo) (pcmk__graph_t *graph, pcmk__graph_action_t *action);
+    int (*pseudo) (pcmk__graph_t *graph, pcmk__graph_action_t *action);
```

Comment:
All these functions return pcmk_rc_ok all the time, so do they even need a return value?